Post-processing triggering condition (#1723)
codemagic-ci-cd/codemagic-docs: content/yaml-publishing/app-store-connect.md

Note that Codemagic does not send status updates on the post-processing step. You can check the build log for the status of post-processing or check your email for updates from App Store Connect.

Post-processing does not consume any build minutes.

> 
> {{<notebox>}}
> **Note:** Post-processing step is triggered only when release notes are found in the project's working directory with the "whatsNew" keyword in it:
>  ``` 
>   [
>     {
>         "language": "en-US",
>         "text": "Customized text",
>         "whatsNew": "Customized description",
>     }
>   ]
>   ```
> {{</notebox>}}

Reviewer: ```suggestion

{{<notebox>}}
**Note:** Post-processing triggered by the presence of release notes will only occur if the release notes are found in the project's working directory and contain the "whatsNew" keyword:
 ``` 
  [
    {
        "language": "en-US",
        "text": "Customized text",
        "whatsNew": "Customized description",
    }
  ]
  ```
{{</notebox>}}
```

Author: done
